test-tests/shared/src/test/scala/zio/test/environment/SystemSpec.scala

```diff
@@ -0,0 +1,76 @@
+package zio.test.environment
+
+import zio.test.Assertion._
+import zio.test.environment.TestSystem.Data
+import zio.test.{ ZIOBaseSpec, suite, _ }
+
+object SystemSpec
```

Comment:
Same comment as on `ConsoleSpec` that you could just access the `TestSystem` in the environment instead of making your own if you just using the default data.